robotics/sensor/bms/BatteryPackStatus.0.1.uavcan

```diff
@@ -0,0 +1,114 @@
+#
+# pragma: no-bit-length-limit
+#
+
+# The network-synchronized timestamp of the moment where the values contained in this message have been sampled.
+uavcan.time.SynchronizedTimestamp.1.0 timestamp
+
+# --------------------------  ESTIMATES  --------------------------
+
+void1
+
+# The instant SoC of the pack, ranges from 0 to 100. Mandatory.
+# The methods of estimation are implementation-defined.
+uint7 state_of_charge  # [percent]
+
+@assert _offset_ % 32 == {0}
+
+# Total instant pack voltage. Low-pass filtering should be applied to avoid aliasing effects.
+uavcan.si.unit.voltage.Scalar.1.0 voltage
+
+# Total instant pack current. Low-pass filtering should be applied to avoid aliasing effects.
+# Positive current flows from the battery to the load (discharging).
+# Negative current flows into the battery (charging).
+uavcan.si.unit.electric_current.Scalar.1.0 current
+
+# A pessimistic estimate of the amount of energy that can be reclaimed from the battery in its current state.
+# This may be dependent on the state of charge (SoC), state of health (SoH), temperature, load profile, etc.
+# Negative values may be reported to indicate overdischarge.
+#
+# The instant power consumption can be computed directly from the voltage and current measurements, or as
+# the time derivative of estimated energy (dE/dt). The latter approach takes into accout other environmental
+# factors that may affect the amount of reclaimable energy, so it may be preferable. Having the instant power,
+# the time to depletion is estimated as ((full_energy - energy) / power). When charging, the remaining time to
+# full charge can be found similarly.
+uavcan.si.unit.energy.Scalar.1.0 energy
+
+# A pessimistic estimate of the amount of energy that can be reclaimed from a fully charged battery
+# under the current conditions (SoH, temperature, load profile, etc).
+uavcan.si.unit.energy.Scalar.1.0 full_energy
+
+# If there are multiple temperature sensors, reduction is implementation-defined. NaN if not defined.
+float16 temperature  # [kelvin]
+
+@assert _offset_ % 8 == {0}
+
+# --------------------------  STATUS FLAGS  --------------------------
+
+# Whether the charging is in progress is observable though the reported instant current: a negative value
+# indicates that the battery is being charged. Time to full charge can be derived from the energy estimate.
+bool charger_connected
+
+# The temperature of at least one cell/section of the battery is above or below the safe operating limits.
+# Note that the flags are not mutually exclusive.
+bool temperature_hot
+bool temperature_cold
+
+# The charge or discharge rate exceeds the safe operating limits.
+bool overload
+
+# The cells require rebalancing. This may be an indicator of deterioration of the battery.
+bool disbalance
+
+# The battery requires other type of maintenance (not balancing).
+# Use vendor-specific interfaces to obtain further information.
+bool needs_service
+
```

Comment:
The BMS has started the balancing process. 
bool balancing

Reply:
Added.